appengine/flexible/CloudSql/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+}
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - RedisCache</title>

```

Comment:
Small nit : remove "- RedisCache"

Reply:
runTests.ps1 failed with "FAIL Found "Visitor Log" within the selector "title". 
Doing a view source on the deployed app's index page shows:

"<title> - RedisCache</title>"

Updating this line in test.js : 
"this.test.assertSelectorHasText('title', 'Visitor Log');"
to be
"this.test.assertSelectorHasText('H2', 'Visitor Log');"

allows the test to pass. So it appears that '@ViewData["Title"]' isn't being populated.